### ISSUE TITLE
#Request: Show auto hidden bookmark and main toolbar on active tab click

### ISSUE THREAD

**Issue by MachineThatGoesP1NG** [MrOtherGuy/firefox-csshacks#358]
Hello I'm using your CSS script "autohide_bookmarks_and_main_toolbars.css" and, I was wondering if it's possible to make it so that the bookmark and main toolbar only drop down on clicking the active tab, instead of on hover. I'd still prefer for them to hide automatically, but when I go to switch tabs with the mouse, the auto drop down kind of makes for a clunky experience. 

Note: I've also made the hover time for the mouse longer to avoid this but I think the above option would work better.

You're doing good stuff for the world and I appreciate all your work!

**Comment by MrOtherGuy** [comment]
There is [click_selected_tab_to_focus_urlbar.css](https://github.com/MrOtherGuy/firefox-csshacks/blob/master/chrome/click_selected_tab_to_focus_urlbar.css) which might still work. It's super hacky but probably closest you can get for the behavior you want. Try how that works with autohide_bookmarks_and_main_toolbars.css. 

If it works at all, then you could set toolbars to never appear on hover. Effectively only showing toolbars when urlbar is focused.

**Comment by MachineThatGoesP1NG** [comment]
> There is [click_selected_tab_to_focus_urlbar.css](https://github.com/MrOtherGuy/firefox-csshacks/blob/master/chrome/click_selected_tab_to_focus_urlbar.css) which might still work. It's super hacky but probably closest you can get for the behavior you want. Try how that works with autohide_bookmarks_and_main_toolbars.css.
> 
> If it works at all, then you could set toolbars to never appear on hover. Effectively only showing toolbars when urlbar is focused.

Ok I'll give that a go. Do you know the specific lines I would replace or swap? I hate to ask him so much but I am terrible at Css

**Comment by MrOtherGuy** [comment]
No, actually that isn't going to work with how autohide_bookmarks_and_main_toolbars.css does things.

I made a quick mockup hack [here](https://gist.github.com/MrOtherGuy/e627ebfd3c3313848a1f882cc42477b3) about how this could work, but I don't plan on supporting it.
